daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -150,16 +154,23 @@ private[archive] class DecodeV1(minor: LV.Minor) {
       case Right(x) => x
     }
 
-  private[archive] def decodeInternedTypes(
+  private[archive] def decodeInternedTypesForTest( // test entry point
       env: Env,
       lfPackage: PLF.Package,
   ): IndexedSeq[Type] = {
+    runWork(decodeInternedTypes(env, lfPackage))
+  }
```

Comment:
why not moving that down below, under the new line 
```
// decode*ForTest -- test entry points
```

Reply:
because it is not a member of the `Env` subclass

Reply:
make sens.  Thanks.